### DISCUSSION TITLE
Support for STM32F405VG?

### DISCUSSION THREAD

**Discussion by electroncraz** [stm32duino/Arduino_Core_STM32#2510] [General]
There is compatibility with the 64 pin version STM32F405RG, but when I use the RG support for the 100pin VG, I have to use hardware registers to get pin states to change. Majority of the extra pins are simply GPIO and Timer channels. Is there future support for the STM32F405VG?

**Comment by fpistm** [comment]
Hi @electroncraz 
You can add it following this wiki: https://github.com/stm32duino/Arduino_Core_STM32/wiki/Add-a-new-variant-%28board%29
Linked folder: https://github.com/stm32duino/Arduino_Core_STM32/tree/main/variants/STM32F4xx/F405VGT_F415VGT
